src/Symfony/Component/Security/Core/Authorization/Voter/VoterInterface.php

```diff
@@ -31,10 +31,10 @@
      * ACCESS_GRANTED, ACCESS_DENIED, or ACCESS_ABSTAIN.
      *
      * @param TokenInterface $token      A TokenInterface instance
-     * @param object|null    $object     The object to secure
+     * @param mixed          $subject    The subject to secure
```

Comment:
this is a BC break for existing voters, as they must now be written in a way accepting any kind of value here, not just ``object|null``. So if we do it, we must document it as a BC break.